build.gradle

```diff
@@ -80,6 +80,19 @@ dependencies {
     testImplementation "org.mockito:mockito-core:3.+"
 }
 
+// When using the gcloud functions deploy --source option to specify the location of the cloud functions source code
```

Comment:
Has AppSec approved the use of a --source option? Does this option mean that someone could run a cloud function from any source that is reachable online?

Reply:
Updated the question "can it pull from any URL it can reach?" to "can it pull from any GCS bucket it can reach?".